README.md

```diff
@@ -174,6 +174,49 @@ yarn build:integration
 yarn test:integration
 ```
 
+## Branching Model and Releases
+
+<!-- TODO: explain about changesets + how we do npm publishing + docker publishing -->
+
+### Active Branches
+
+| Branch          | Status                                                                           |
+| --------------- | -------------------------------------------------------------------------------- |
+| [master](https://github.com/ethereum-optimism/optimism/tree/master/)                   | Does not accept PRs. Changes are force-pushed from `develop` when we intend to deploy to mainnet.            |
```

Comment:
Rewording suggestion:

>Changes are force-pushed from `develop`

->Changes are merged from `develop` when we release.

Reply:
Liam asked that I explicitly say "force pushed" because that's apparently what we're doing right now. I do think we should be merging instead but better to reflect the reality for now IMO.

Reply:
Liam just confirmed that we should be merging and not force pushing. I will update the readme to reflect that.